### PR TITLE
Feature/cart update

### LINKED ISSUE #23: Fix functionality so that UI is not one behind state

### LINKED ISSUE #42: Cart page and Cart Api loading
As a user, I should be able to...
- [x] Add an item to the cart on the image page and have the quantity reflect that on the cart page.

### PR DESCRIPTION
**Pull request by edwardkrupicka**
### What'd ya do to the code?

Refactored the fetch requests so our UI works async with our API. Big changed I did was I moved the add to cart function to the app.js where it set state for cart in app.js, the cart state is then passed down to cart.js.

### Select One Thing
- [x] This broke nothing
- [ ] This broke something
- [ ] This broke everything

### Type of change
- [x] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Refactor(DRY-ing up code, removing comments etc.)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Checklist:
- [ ] If this code needs to be tested, all tests are passing
- [x] The code runs locally
- [ ] There are comments where clarification/ organization is needed
- [ ] The code is DRY. If it's not, I tried my best
- [x] I reviewed my code before pushing

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -25,33 +31,72 @@ const App = () => {
...
   const deleteFromFavorites = async (newData) => {
-    await fetch(`http://localhost:3001/api/v1/favorites/${newData.id}`, {
-      method: 'DELETE'
-    })
-      await ((res) => res.json())
-      await fetchFavData()
-      await fetchImageData()
-      .catch(err => console.log(err))
+    const config = {
+      method: 'Delete',
```
Comment (reganlosey) [inline src/App.js:83]: Didn't know method names didn't need to be capitalized? If it works that's good to know for future troubleshooting

**Review by reganlosey — APPROVED** [review]
Cool

**Review by delilahrois — APPROVED** [review]
Looks good, thanks!

**Review by tanya2108 — APPROVED** [review]
Looks good, thank you!

**Pull request merged**